File: SearchAndFindBackend/SearchAndFind.DTO/Saler/SalerAvailableForTenderDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class SalerAvailableForTenderDTO 
    {
        public double Latitude { get; set; }
        public double Length { get; set; }
        public string ShopName { get; set; }
        public string ShopAddress { get; set; }
        public string ShopPhone { get; set; }
        public double Distance { get; set; }
        public string DeviceId { get; set; }
       
    }
}

File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/query/SalerAvailableForTenderDTO.java
package uy.com.searchandfind.query;

/**
 * Created by strat on 24/10/2017.
 */

public class SalerAvailableForTenderDTO {
    private double Latitude ;
    private double Length ;
    private String ShopName ;
    private String ShopAddress ;
    private String ShopPhone ;
    private double Distance ;

    public double getLatitude() {
        return Latitude;
    }

    public void setLatitude(double latitude) {
        Latitude = latitude;
    }

    public double getLength() {
        return Length;
    }

    public void setLength(double length) {
        Length = length;
    }

    public String getShopName() {
        return ShopName;
    }

    public void setShopName(String shopName) {
        ShopName = shopName;
    }

    public String getShopAddress() {
        return ShopAddress;
    }

    public void setShopAddress(String shopAddress) {
        ShopAddress = shopAddress;
    }

    public String getShopPhone() {
        return ShopPhone;
    }

    public void setShopPhone(String shopPhone) {
        ShopPhone = shopPhone;
    }

    public double getDistance() {
        return Distance;
    }

    public void setDistance(double distance) {
        Distance = distance;
    }
}
